doc/source/building/windows.rst

```diff
@@ -168,90 +281,115 @@ the following in your build log:**
 
    FOUND:
       libraries = ['openblas']
-      library_dirs = ['C:\opt\64\lib']
-      language = c
+      library_dirs = ['C:\...........\lib']
+      language = f77
       define_macros = [('HAVE_CBLAS', None)]
 
+We'll get back to this below how to check.
```

Comment:
> We'll get back to this below how to check.

Does this mean that later in the document it will explain how to check the log? Why not right here?

Reply:
This is done now.